src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -2,7 +2,7 @@
   margin: 8px;
   font-size: 14px;
   max-width: 20%;
-  width: 17%; /* depends on number of filters*/
+  width: 13%; /* depends on number of filters*/
```

Comment:
Currently the dropdown filters are already quite cramp. Perhaps decrease the width of the search bar instead?